.Rbuildignore

```diff
@@ -28,3 +28,5 @@ SMG.md
 ^inst/CountryTemplate\.R$
 ^inst/paper$
 ^inst/examples$
+^inst/github_workflow_template\.yaml$
+^tests/testthat/test-github_workflow_present\.R$
```

Comment:
TBH I'm not really sure if thats working I just copied the format for the other stuff (`CountryTemplate.R`)

Reply:
Yes me neither. CountryTemplate isn't used in the tests (I think) and so not likely to be a problem. Maybe just leave in especially given suggestion this could be a nice package function.